notebooks/224-vision-image-colorization/224-vision-image-colorization.ipynb

```diff
@@ -0,0 +1,511 @@
+    "MODEL_PATH = f\"{MODEL_DIR}/colorization-v2/{PRECISION}/{MODEL_NAME}.xml\"\n",
+    "DATA_DIR = \"data\"\n",
+    "DEVICE=\"CPU\""
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "4cf87170-fd22-4fcf-b687-51b404be0db4",
+   "metadata": {},
+   "source": [
+    "## Download the model\n",
+    "\n",
+    "`omz_downloader` downloads model files from online sources and, if necessary, patches them to make them more usable with Model Convertor.\n",
+    "\n",
+    "In our case `omz_downloader` downloads the checkpoint and pytorch model of [colorization-v2](https://github.com/openvinotoolkit/open_model_zoo/blob/master/models/public/colorization-v2/README.md) from [Open Model Zoo](https://github.com/openvinotoolkit/open_model_zoo/blob/master/models/public/index.md) and saves it under `MODEL_DIR` as specified in above configuration"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 3,
+   "id": "679a06d8-d0ec-440f-ae7a-afc4ebe841e6",
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "################|| Downloading colorization-v2 ||################\n",
+      "\n",
+      "========== Downloading /mnt/Backup/my_works/openvino_notebooks/notebooks/224-vision-image-colorization/public/colorization-v2/ckpt/colorization-v2-eccv16.pth\n",
+      "... 100%, 125953 KB, 7492 KB/s, 16 seconds passed\n",
+      "\n",
+      "========== Downloading /mnt/Backup/my_works/openvino_notebooks/notebooks/224-vision-image-colorization/public/colorization-v2/model/__init__.py\n",
+      "... 100%, 0 KB, 93 KB/s, 0 seconds passed\n",
+      "\n",
+      "========== Downloading /mnt/Backup/my_works/openvino_notebooks/notebooks/224-vision-image-colorization/public/colorization-v2/model/base_color.py\n",
+      "... 100%, 0 KB, 2083 KB/s, 0 seconds passed\n",
+      "\n",
+      "========== Downloading /mnt/Backup/my_works/openvino_notebooks/notebooks/224-vision-image-colorization/public/colorization-v2/model/eccv16.py\n",
+      "... 100%, 4 KB, 15317 KB/s, 0 seconds passed\n",
+      "\n",
+      "========== Replacing text in /mnt/Backup/my_works/openvino_notebooks/notebooks/224-vision-image-colorization/public/colorization-v2/model/__init__.py\n",
+      "========== Replacing text in /mnt/Backup/my_works/openvino_notebooks/notebooks/224-vision-image-colorization/public/colorization-v2/model/__init__.py\n",
+      "========== Replacing text in /mnt/Backup/my_works/openvino_notebooks/notebooks/224-vision-image-colorization/public/colorization-v2/model/eccv16.py\n",
+      "\n"
+     ]
+    }
+   ],
+   "source": [
+    "!omz_downloader --name colorization-v2"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "93715d09-a6b9-4af5-b129-462c0cb9fc27",
+   "metadata": {},
+   "source": [
+    "## Convert the model to OpenVINO IR\n",
+    "\n",
+    "`omz_converter` converts the models that are not in the OpenVINO™ IR format into that format using Model Optimizer.\n",
```

Comment:
Same for the converter. Please don't use bash.

![image](https://user-images.githubusercontent.com/10940214/165588319-2b7ad02f-be03-44de-a514-f71bb8a14c6c.png)